<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Demo Test Suite" duration-ms="2845" started-at="2021-03-30T22:08:31 EEST" finished-at="2021-03-30T22:08:33 EEST">
    <groups>
    </groups>
    <test name="TC_01_Domain" duration-ms="2845" started-at="2021-03-30T22:08:31 EEST" finished-at="2021-03-30T22:08:33 EEST">
      <class name="com.demo.tests.TestCase_01">
        <test-method status="PASS" signature="cleanReportData()[pri:0, instance:com.demo.tests.TestCase_01@1e8b7643]" name="cleanReportData" is-config="true" duration-ms="12" started-at="2021-03-30T22:08:30 EEST" finished-at="2021-03-30T22:08:31 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanReportData -->
        <test-method status="FAIL" signature="add_new_basket(java.lang.reflect.Method)[pri:0, instance:com.demo.tests.TestCase_01@1e8b7643]" name="add_new_basket" duration-ms="2480" started-at="2021-03-30T22:08:31 EEST" description="API" finished-at="2021-03-30T22:08:33 EEST">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.demo.tests.TestCase_01.add_new_basket(java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.demo.scripts.CreateNewBasket.newBasket(CreateNewBasket.java:92)
at com.demo.tests.TestCase_01.add_new_basket(TestCase_01.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.TestNG.privateMain(TestNG.java:1336)
at org.testng.TestNG.main(TestNG.java:1304)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_new_basket -->
        <test-method status="PASS" signature="generateReport(org.testng.ITestContext, org.testng.ITestResult)[pri:0, instance:com.demo.tests.TestCase_01@1e8b7643]" name="generateReport" is-config="true" duration-ms="9" started-at="2021-03-30T22:08:33 EEST" finished-at="2021-03-30T22:08:33 EEST">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@48cd9a2c]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[TestResult name=add_new_basket status=FAILURE method=TestCase_01.add_new_basket(java.lang.reflect.Method)[pri:0, instance:com.demo.tests.TestCase_01@1e8b7643] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReport -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.demo.tests.TestCase_01@1e8b7643]" name="endReport" is-config="true" duration-ms="334" started-at="2021-03-30T22:08:33 EEST" finished-at="2021-03-30T22:08:33 EEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.demo.tests.TestCase_01 -->
    </test> <!-- TC_01_Domain -->
  </suite> <!-- Demo Test Suite -->
</testng-results>
